releases/v1.1-alpha.20170817.md

```diff
@@ -0,0 +1,73 @@
+---
+title: What&#39;s New in v1.1-alpha.20170817
+toc: false
+summary: Additions and changes in CockroachDB version v1.1-alpha.20170817
+---
+
+## Aug 17, 2017
+
+Get future release notes emailed to you:
+
+<div class="hubspot-install-form install-form-1 clearfix">
+    <script>
+        hbspt.forms.create({
+            css: '',
+            cssClass: 'install-form',
+            portalId: '1753393',
+            formId: '39686297-81d2-45e7-a73f-55a596a8d5ff',
+            formInstanceId: 1,
+            target: '.install-form-1'
+        });
+    </script>
+</div>
+
+### Downloads
+
+<div id="os-tabs" class="clearfix">
+    <a href="https://binaries.cockroachdb.com/cockroach-v1.1-alpha.20170817.darwin-10.9-amd64.tgz"><button id="mac" data-eventcategory="mac-binary-release-notes">Mac</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v1.1-alpha.20170817.linux-amd64.tgz"><button id="linux" data-eventcategory="linux-binary-release-notes">Linux</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v1.1-alpha.20170817.windows-6.2-amd64.zip"><button id="windows" data-eventcategory="windows-binary-release-notes">Windows</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v1.1-alpha.20170817.src.tgz"><button id="source" data-eventcategory="source-release-notes">Source</button></a>
+</div>
+
+### SQL Language Changes
+
+- New statements `PAUSE JOB`, `RESUME JOB`, and `CANCEL JOB` can be used to control long-running jobs (schema changes and enterprise backup/restore). [#17434](https://github.com/cockroachdb/cockroach/pull/17434)
```

Comment:
According to cockroachdb/cockroach#16018, these commands currently only work with backup/restore, not schema changes.